app/images/icons/no-result-magnifier.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M37.7506 19.624C37.7514 26.9556 33.3354 33.5658 26.562 36.3719C19.7886 39.1779 11.9918 37.6273 6.80756 32.443C1.6233 27.2588 0.0726455 19.462 2.87872 12.6886C5.68479 5.91515 12.2949 1.49919 19.6266 1.5C24.4334 1.5 29.0433 3.40949 32.4422 6.8084C35.8411 10.2073 37.7506 14.8172 37.7506 19.624Z" stroke="#5C5589" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M13.1367 13.136L25.8647 25.864" stroke="#5C5589" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M13.1367 25.864L25.8647 13.136" stroke="#5C5589" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M32.4434 32.4399L46.5014 46.4999" stroke="#5C5589" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
```

Comment:
All the colours in here (e.g. `#5C5589`) should be changed to `black`. All icons should be black and should have a `filter-xxx` applied when they're used.